py/src/braintrust/logger.py

```diff
@@ -4291,6 +4297,27 @@ def render_prompt_params(params: Dict[str, Any], args: Any) -> Dict[str, Any]:
     return {**params, "response_format": {**response_format, "json_schema": {**json_schema, "schema": parsed_schema}}}
 
 
+def render_mustache(template: str, data: Any, *, strict: bool = False, renderer: Optional[Callable[..., Any]] = None):
+    if renderer is None:
+        renderer = chevron.render
+
+    if not strict:
+        return renderer(template, data=data)
+
+    # Capture stderr to check for missing keys
+    stderr_capture = io.StringIO()
+    with contextlib.redirect_stderr(stderr_capture):
```

Comment:
opted with this to defer as much as we can to the actual template rendering library